sopel/config/core_section.py

```diff
@@ -526,6 +526,57 @@ class CoreSection(StaticSection):
     .. versionadded:: 7.0
     """
 
+    flood_max_wait = ValidatedAttribute('flood_max_wait', float, default=2)
+    """How much time to wait at most when flood protection kicks in.
+
+    :default: ``2``
+
+    This is equivalent to the default value:
+
+    .. code-block:: ini
+
+        flood_max_wait = 2
+
+    .. seealso::
+
+        The :ref:`Flood Prevention` chapter to learn what each flood-related
+        setting does.
+
+    .. versionadded:: 7.1
+    """
+
+    flood_penalty_ratio = ValidatedAttribute('flood_penalty_ratio',
+                                             float,
+                                             default=1.4)
+    """Ratio of the message length used to compute the added wait penalty.
+
+    :default: ``1.4``
+
+    Message that are longer than :attr:`flood_text_length` will get an added
+    wait penalty (in seconds) that will be computed like this::
+
+        overflow = max(0, (len(text) - flood_text_length))
+        rate = flood_text_length * flood_penalty_ratio
+        penalty = overflow / rate
+
+    .. note::
+
+        If the penalty ratio is 0 (or less), this penalty will be disabled.
```

Comment:
```suggestion
        If the penalty ratio is 0, this penalty will be disabled.
```

Per line note in `configuration.rst` docs.